refinery/data_set_manager/views.py

```diff
@@ -1148,6 +1148,23 @@ def get_object(self, uuid):
             logger.error(e)
             raise APIException("Multiple objects returned.")
 
+    def get_first_annotated_node_from_solr_name(self, solr_name, node):
```

Comment:
I think this would be more appropriate outside of the View class, maybe in a `utils.py`. Having this logic within the view makes it harder to test, and it doesn't utilize the reference to `self`.

For example, you wouldn't have to instantiate `NodeViewSet()` and give it a `request` just to test it anymore.

Reply:
Sure.